API_Extended_Document/util/config.yml

```diff
@@ -1,6 +1,6 @@
 ordbms: postgresql
 user: postgres
-password: password
+password: postgres
 host: localhost
 port: 5432
-dbname: extendedDoc
```

Comment:
Hummm, is there a simple way for no "commiting" such passwords (usual methods include reading an environment variable at run time of the script)